crates/pallet-feeds/src/mock.rs

```diff
@@ -0,0 +1,72 @@
+use crate as pallet_feeds;
```

Comment:
And the API tests are fine to stay inside per crate, kind of odd to move them into integration tests. For me, these are mere API tests(unit tests) and not integration tests.

Since we have come to this topic, just want to say that my understanding of the integration tests in this field is that we create a node with all components(transaction pool, DB, etc) to test the whole business process(submit some extrinsics, build blocks, sync blocks, etc) properly. So the current integration tests in subspace are not the typical integration tests I've learned IMHO, they are just API tests.

Reply:
Yes, it is fine, I'm just saying that we care about public API, in which case what is considered "integration" test in Rust helps to make sure we don't access something that is not supposed to be accessed.